javascripts/main.js

```diff
@@ -154,7 +154,7 @@ define([
       });
 
     projectsPanel
-      .find('ul.popular-tags')
+      .find('div.popular-tags')
```

Comment:
I can't spot what's caused this change but clicking this links no longer causes the page to filter on the tag.

If I vitsit https://up-for-grabs.net/# and click on the `javascript` tag I see the URL change to `https://up-for-grabs.net/#/filters?tags=javascript` and the form shows that `javascript` is in the list:

<img width="714" src="https://user-images.githubusercontent.com/359239/197286134-eba2357a-525b-4146-a961-f892e35ea186.png">

However if I visit the staging site for this PR https://deploy-preview-3460--up-for-grabs-test-bench.netlify.app/#/ and click the `javascript` nothing happens. 

I think this needs to change back to `ul` as this appears to be the click handlers for the tags:

```suggestion
      .find('ul.popular-tags')
```

Reply:
ooof that's embarrassing 🤦‍♀️. sorry about that, should be all set now